app/src/main/java/mozilla/lockbox/support/DataStoreSupport.kt

```diff
@@ -42,25 +45,70 @@ class FixedDataStoreSupport(values: List<ServerPassword>? = null) : DataStoreSup
 
 /**
  * Creates a test ServerPassword item
+ * Some functionality inspired by FxA 'upload_fake_passwords.py'
+ * https://gist.github.com/rfk/916d9ca684f862b1c1030c685a5a4d19
  */
 internal fun createDummyItem(idx: Int): ServerPassword {
-    val rng = Random()
-    val pos = idx + 1
-    val pwd = "AAbbcc112233!"
-    val host = "https://$pos.example.com"
+    val random = Random()
+    val pwd = createRandomPassword()
+    val host = createHostname()
+    val user = createUserId()
     val created = Date().time
     val used = Date(created - 86400000).time
     val changed = Date(used - 86400000).time
 
     return ServerPassword(
-            id = "0000$idx",
-            hostname = host,
-            username = "someone #$pos",
-            password = pwd,
-            formSubmitURL = host,
-            timeCreated = created,
-            timeLastUsed = used,
-            timePasswordChanged = changed,
-            timesUsed = rng.nextInt(100) + 1
+        id = "0000$idx",
+        hostname = host,
+        username = user,
+        password = pwd,
+        formSubmitURL = host,
+        timeCreated = created,
+        timeLastUsed = used,
+        timePasswordChanged = changed,
+        timesUsed = random.nextInt(100) + 1
     )
 }
+
+internal fun createRandomPassword(): String {
+    return UUID.randomUUID().toString().substring(0..20)
+}
+
+internal fun createHostname(): String {
+    val protocolChoices = listOf(
+        "https://www.",
+        "http://www.",
+        "https://",
+        "https://accounts."
+    )
+    val domainChoices = listOf(
+        ".com",
+        ".net",
+        ".org",
+        ".co.uk"
+    )
+    var protocol = protocolChoices[Random().nextInt(protocolChoices.size)]
+    var domain = domainChoices[Random().nextInt(domainChoices.size)]
+
+    var hostnameLength = getRandomInRange(8, 20)
+    val hostname =
+        buildSequence {
+            val r = Random(); while (true) yield(r.nextInt(24))
```

Comment:
related to "67 vs 65" below, why use `24` instead of `26`?

Reply:
same as other. changing this